hack/errata-check.sh

```diff
@@ -0,0 +1,262 @@
+      exit 1
+      ;;
+  esac
+  print-label "WMCO Version" "v$WMCO_VERSION"
+}
+
+function check-upstream-status {
+  # Checks if the local repo is up to date with
+  local branch=$1
+  print-label "Branch" "$branch"
+  local local_head
+  local upstream_head
+  local_head=$(git rev-parse "$branch")
+  upstream_head=$(git ls-remote upstream -h "refs/heads/$branch" | awk '{ print $1 }')
+  if [ "$local_head" != "$upstream_head" ]; then
+    print-error "Local branch $branch is not up to date with upstream"
+    exit 1
+  fi
+}
+
+function curl-json {
+  local url=$1
+  local result
+  if ! result=$(curl --silent --negotiate --user : "$url"); then
+    print-error "Unable to curl server.  Connected to VPN?"
+    exit 1
+  fi
+  echo -E "$result"
+}
+
+function add-build-tags {
+  # Add a tag for the build nvr (wmco-container-X.Y.Z-n)
+  print-info "Adding build tag:"
+  WMCO_TAG=wmco-container-${BUILD_NVR:42}
+  if git show-ref --tags "refs/tags/$WMCO_TAG"; then
+      printf "${GREEN}%s${NORMAL}\n" "Tag for $WMCO_TAG already exists"
+  else
+      printf "${YELLOW}%s${NORMAL}\n" "Tagging WMCO hash $GITHUB_HASH as $WMCO_TAG"
+      git tag "$WMCO_TAG" "$GITHUB_HASH"
+  fi
+}
+
+function get-commits {
+  local start_hash=$1
+  local end_hash=$2
+  mapfile -t MERGE_COMMITS < <(git rev-list --format=oneline --merges "$start_hash...$end_hash")
+}
+
+function list-commits {
+  local color=$1
+  local -r format="%-40s %-4s %-65.65s %-13s %s\n"
+  printf "${BLUE}"
+  printf "$format" "Commit Hash" "PR#" "PR Title" "Jira ID" "Errata Status"
+  printf "%s %s %s %s %s\n" "$(printf "=%.0s" {1..40})" "$(printf "=%.0s" {1..4})" "$(printf "=%.0s" {1..65})" "$(printf "=%.0s" {1..13})" "$(printf "=%.0s" {1..13})"
+  printf "$color"
+  for COMMIT in "${MERGE_COMMITS[@]}"
+  do
+    JIRA_ID=""
+    ATTACHED=""
+    COMMIT_HASH=$(echo "$COMMIT" | awk '{ print $1 }')
+    PR_NUM=$(echo "$COMMIT" | awk '{ print substr($5,2) }')
+    PR_TITLE=$(gh pr view "$PR_NUM" --json title | jq -r .title)
+    JIRA_ID=$(echo $PR_TITLE | grep -Eo '(WINC|OCPBUGS)-[0-9]*') || true
+    if [ ! -z "$JIRA_ID" ]; then
+      if [[ "$ERRATA_JIRA_IDS" == *"$JIRA_ID"* ]]; then
+        ATTACHED="${GREEN}Attached${color}"
+      else
+        ATTACHED="${YELLOW}Not Attached${color}"
+      fi
+    fi
+    printf "$format" "$COMMIT_HASH" "$PR_NUM" "$PR_TITLE" "$JIRA_ID" "$ATTACHED"
+  done
+  printf "${NORMAL}"
+}
+
+###############
+# Main script #
+###############
+verify-args "$@"
+
+TARGET_BRANCH=$1
+set-branch-vars "$TARGET_BRANCH"
+check-upstream-status "$TARGET_BRANCH"
+
+# Query Errata Tool for the active errata for the desired release version
+ERRATA_SEARCH=$(curl-json "$ERRATA_API_URL/search?product%5B%5D=79&show_state_IN_PUSH=1&show_state_NEW_FILES=1&show_state_PUSH_READY=1&show_state_QE=1&show_state_REL_PREP=1&synopsis_text=Windows+Containers+$WMCO_VERSION")
+
+# Parse errata search result for errata ID
+if ! ERRATA_ID=$(echo -E "$ERRATA_SEARCH" | jq '.data | .[] | .id') ; then
```

Comment:
I see a parse error with an invalid kerberos ticket:
```
windows-machine-config-operator   release-4.10 → upstream {4}  1  errata-check.sh release-4.10
WMCO Version        = v5
Branch              = release-4.10
parse error: Invalid numeric literal at line 1, column 10

ERROR: Unable to parse errata search results.  Valid kerberos ticket?
```
Not a big deal but I think you can error check `ERRATA_ID` before passing it to `jq`. 

PS: this not a blocker for merging this.

Reply:
Added a [check](https://github.com/openshift/windows-machine-config-operator/pull/1462/files#diff-860bfe2660f784c0e166c943676def7106a2c50fcfe353ee6c6c6f48a2c4c95aR150-R153) for `unauthorized` in the response before continuing.